lib/ecto/association.ex

```diff
@@ -555,6 +555,7 @@ defmodule Ecto.Association.Has do
     * `on_replace` - The action taken on associations when schema is replaced
     * `defaults` - Default fields used when building the association
     * `relationship` - The relationship to the specified schema, default is `:child`
+    * `preload_order` - Default `order_by` of the association, used by preload
   """
```

Comment:
I think it would be good to specify that this is used ONLY by preload (will it be used by assoc? if yes, it should be mentioned here too) and not by other queries. Or maybe link to the longer documentation snippet.

Reply:
```diff
-     * `preload_order` - Default `order_by` of the association, used by preload
+     * `preload_order` - Default `order_by` of the association, used only by preload
``
@v0idpwn Do you think adding the `only` would be enough, or we should be even more explicit about that?

Reply:
only lgtm

Reply:
Done, thanks @v0idpwn